3rdparty/clapack/include/lapack.h

```diff
@@ -86,7 +86,7 @@ int dhseqr_(char *job, char *compz, int *n, int *ilo, int *
 
 int disnan_(double *din);
 
-int dlabad_(double *small, double *large);
+int dlabad_(double *_small, double *_large);
```

Comment:
What is wrong? Looks like we have malformed macro somewhere.

Reply:
This is not related to rgbd or 3d modules themselves; this can be extacted to different PR.
But that was a stopper, that's why the fix is here.
[The problem](https://stackoverflow.com/a/27794577) is that `small` is a macro defined in Windows headers.

Reply:
in which file?

`#undef small` can be added there

---

Avoid using of underscore prefixes. Conflicts with them are observed more frequently.

Reply:
Fixed by adding `#undef`